abb_driver/rapid/ROS_stateServer.mod

```diff
@@ -29,7 +29,7 @@ MODULE ROS_stateServer
 ! WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 LOCAL CONST num server_port := 11002;
-LOCAL CONST num update_rate := 0.10;  ! broadcast rate (sec)
+LOCAL CONST num update_rate := 0.01;  ! broadcast rate (sec)

```

Comment:
Does that work? Do we get 100Hz feedback now?

Reply:
Not quite, but it's better than 10Hz -- rostopic hz estimate 83-34Hz.